src/test/scala/com/wix/mysql/EmbeddedMysqlTest.scala

```diff
@@ -143,6 +143,9 @@ class EmbeddedMysqlTest extends IntegrationTest {
         .withScripts(
           aMigrationWith("create table t1 (col1 INTEGER);"),
           aMigrationWith("create table t2 (col1 INTEGER);"))
+        .withCommands(
+          "create table t3 (col1 INTEGER);\n" +
```

Comment:
API allows to pass several commands. In this test I showed that it could be one command (string) with several instructions and a single instruction.